deps/chakrashim/src/inspector/v8-timetravel-agent-impl.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "src/inspector/v8-timetravel-agent-impl.h"
+
+#include "src/inspector/v8-inspector-session-impl.h"
+#include "src/inspector/v8-debugger-agent-impl.h"
+
+#include "src/jsrtinspector.h"
+
+namespace v8_inspector {
+
+V8TimeTravelAgentImpl::V8TimeTravelAgentImpl(
+    V8InspectorSessionImpl* session, protocol::FrontendChannel* frontendChannel,
+    protocol::DictionaryValue* state)
+    : m_session(session),
+      m_frontend(frontendChannel),
+      m_state(state) {}
+
+V8TimeTravelAgentImpl::~V8TimeTravelAgentImpl() {}
+
+bool V8TimeTravelAgentImpl::checkEnabled(ErrorString* errorString) {
+  if (enabled()) {
+    return true;
```

Comment:
I can do that, but the pattern elsewhere is to only touch the variable in cases where there's an error.  This was consistent with the state of the V8 code at the time it was forked.